.yamato/all-urp_performance_boatattack.yml

```diff
@@ -48,7 +48,7 @@ PR_URP_Performance_BoatAttack_trunk:
         UTR_VERSION: "current"
         TEST_FILTER: .*
         BOAT_ATTACK_BRANCH: master
-        BOAT_ATTACK_REVISION: d07745c0d9bc97cde5fc4ae86186d6c2f0c2a16c
+        BOAT_ATTACK_REVISION: 88679d7ebeeae4be30f43ebe88cba830f363803b
```

Comment:
this shouldnt change - it should still be the revision starting with d077

Reply:
good catch, changing it now